modules/utils/src/main/java/io/uhndata/cards/serialize/internal/ImportableProcessor.java

```diff
@@ -94,4 +105,17 @@ public JsonValue processChild(final Node node, final Node child, final JsonValue
         }
         return input;
     }
+
+    @Override
+    public String processPropertyName(final Node node, final Property property, final String input)
+    {
+        try {
+            if (SUBJECT_TYPES_PROPERTY.equals(property.getName())) {
+                return "jcr:reference:" + SUBJECT_TYPES_PROPERTY;
+            }
```

Comment:
I think this should be generic, you can see the type of the property, and if it's reference use the prefix.


```suggestion
            if (property.getType() == PropertyType.REFERENCE) {
                return "jcr:reference:" + property.getName();
            }
```

Also needs an import for `PropertyType`.